examples/nbproject/project.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://www.netbeans.org/ns/project/1">
+    <type>org.netbeans.modules.cnd.makeproject</type>
+    <configuration>
+        <data xmlns="http://www.netbeans.org/ns/make-project/1">
+            <name>CppUTest</name>
+            <c-extensions>c</c-extensions>
+            <cpp-extensions>cpp</cpp-extensions>
+            <header-extensions>h</header-extensions>
+            <sourceEncoding>UTF-8</sourceEncoding>
+            <make-dep-projects/>
+            <sourceRootList>
+                <sourceRootElem>src/CppUTest</sourceRootElem>
+                <sourceRootElem>src/CppUTestExt</sourceRootElem>
+                <sourceRootElem>src/Platforms</sourceRootElem>
+                <sourceRootElem>include</sourceRootElem>
+                <sourceRootElem>examples/AllTests</sourceRootElem>
+                <sourceRootElem>../include/CppUTest</sourceRootElem>
+                <sourceRootElem>../include/CppUTestExt</sourceRootElem>
+                <sourceRootElem>../include/Platforms</sourceRootElem>
+                <sourceRootElem>AllTests</sourceRootElem>
+                <sourceRootElem>ApplicationLib</sourceRootElem>
+                <sourceRootElem>../src/CppUTest</sourceRootElem>
+                <sourceRootElem>../src/CppUTestExt</sourceRootElem>
+                <sourceRootElem>../src/Platforms</sourceRootElem>
+                <sourceRootElem>../tests</sourceRootElem>
+            </sourceRootList>
+            <confList>
+                <confElem>
+                    <name>Debug</name>
+                    <type>1</type>
+                </confElem>
+            </confList>
+            <formatting>
+                <project-formatting-style>false</project-formatting-style>
+            </formatting>
+        </data>
+    </configuration>
+</project>
```

Comment:
That wouldn't be for me to decide ;-). Keep in mind that you might be expected to support this if it is there, e.g. if someone else adds some new files, etc.

For the record, I am using Code::Blocks as an IDE and I keep my project files out of the repository (one level above, where I version them in a different repository. I also use automake a lot; saves me surprises when I push :)